otx/algorithms/classification/tasks/inference.py

```diff
@@ -69,8 +69,12 @@ def __init__(self, task_environment: TaskEnvironment):
         else:
             self._labels = task_environment.get_labels(include_empty=False)
         self._empty_label = get_empty_label(task_environment.label_schema)
+
+        self.task_model_dir = None
+        self.task_pipeline_path = None
         self._multilabel = False
         self._hierarchical = False
+        self._selfsl = False
```

Comment:
Wouldn't this variable be commonly used by other tasks? If you need to use other tasks as well, it would be nice to declare them in Common's `BaseTask`.

Reply:
Actually, that variable is used several times in classification, but just once in segmentation.
Currently, I think it is only for classification.